autoscale/src/main/java/com/sequenceiq/periscope/filter/CloudbreakUserConfiguratorFilter.java

```diff
@@ -34,16 +34,15 @@ public CloudbreakUserConfiguratorFilter(AutoscaleRestRequestThreadLocalService r
 
     @Override
     protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
-        CloudbreakUser cloudbreakUser = authenticatedUserService.getCbUser();
+        CloudbreakUser cbUser = authenticatedUserService.getCbUser();
         Long workspaceId = UNINITIALIZED_WORKSPACE_ID;
```

Comment:
I'm not familiar with the UNINITIALIZED_WORKSPACE_ID and why it would have value -1.
But if you wish to call a CB API directly, you should always use workspaceID 0.
That will default to the tenant default workspace which should be already existing.

Reply:
ack. as discussed will default cluster to tenant scope and remove workspace related ids.